src/Simulation/QsharpCore/Core.qs

```diff
@@ -30,6 +30,21 @@ namespace Microsoft.Quantum.Core {
     @Attribute()
     newtype Deprecated = (NewName : String);
 
+    /// # Summary
+    /// Compiler-recognized attribute used to mark a callable with the runtime capabilities it requires.
+    ///
+    /// # Input
+    /// ## Level
+    /// The name of the runtime capability level required by the callable.
+    /// 
+    /// # Remarks
+    /// The valid capability level names, in order of increasing capabilities, are:
+    ///
+    /// 1. QPRGen0
```

Comment:
Is there any documentation available for what features are included in each of these capability levels?

Reply:
I added descriptions to the doc comment here.